.github/workflows/python-app.yml

```diff
@@ -1,4 +1,4 @@
-name: build status
+name: Build Status
    
 on:
```

Comment:
Why the `cron` job?

Reply:
This is a way to make github testing once a week the build (always sunday at 00:00). We can just eliminate it if you prefer.